docs/user/build-customization.rst

```diff
@@ -104,6 +105,45 @@ To avoid this, it's possible to unshallow the clone done by Read the Docs:
          - git fetch --unshallow
 
 
+Skip build based on a condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There may be situations where you want to skip a build that was automatically triggered when someone on your team pushed to the repository.
+Skipping builds will allow you to speed up review times and also help us reduce server costs and ultimately our environmental footprint.
+Consider the following scenarios:
+
+* the build depends on an external situation that's not met yet
+* there were no changes on the documentation files
+
+In these scenarios, you can skip the build by executing a custom command that checks for that particular condition
+and exits with code ``439`` to skip it, or ``0`` to continue building the documentation normally.
+If any of the commands return this particular exit code,
+Read the Docs will stop the build immediately,
+mark it as "Cancelled",
+and communicate to your Git platform (GitHub/GitLab) that the build succeeded (green tick ✅) so the pull request is in a mergeable state.
+
+Here is an example that skip build from pull requests when there are no changes to the ``docs/`` folder compared to the ``origin/main`` branch.
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+
+   version: 2
+   build:
+     os: "ubuntu-22.04"
+     tools:
+       python: "3.11"
+     jobs:
+       post_checkout:
+         # Skip building pull requests when there aren't changed in the docs directory.
+         # `--quiet` exits with a 1 when there **are** changes,
+         # so we invert the logic with a !
+         #
+         # If there are no changes (exit 0) we force the command to return with 439.
+         # This is a special exit code on Read the Docs that will cancel the build immediately.
+         - if [ $READTHEDOCS_VERSION_TYPE = "external" ]; then ! git diff --quiet origin/main -- docs/ && exit 439; fi
```

Comment:
Hello. Would be nice to have example to show to do implement "skip RTD build if there is a [ci skip], [skip ci], [skip rtd], or [rtd skip]" in the last commit message. Thanks!

Reply:
Thanks! I am excited to try it out when it is available. 😸

Reply:
You can already test this 😉 , but keep reading...

... the only difference with this PR is that (when merged and deployed) it:

* will show a nicer error message instead of saying that your build is broken
* will send "Success" (green tick ✅ ) to GitHub when skipping the build instead "Failed" (red cross ❌ )

The links that I've shared are from production already 😎